apps/editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -533,6 +532,9 @@ export const editorStore: StateCreator<
       variables: {
         teamId,
       },
+      // Flows are modified via REST API requests, not via the Apollo client
+      // Always get an up to date list when showing the page
+      fetchPolicy: "network-only",
```

Comment:
There are many better ways of handling this, such as manual invalidations on these operations.

Parked for now as this PR is already too big!